misc/formatter/modules/formatter-core/src/test/resources/misc/ranges/range-test-config.json

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
I feel this structure can be improved. What we need is a 'list' of filenames and their ranges. The best data structure to represent a list is a json array, not a json object. That will make it easier to iterate as well.

Reply:
Implemented as an array within the JSON file.